pkg/cloudprovider/providers/openstack/openstack.go

```diff
@@ -595,29 +583,20 @@ func (os *OpenStack) volumeService(forceVersion string) (volumeService, error) {
 
 	switch bsVersion {
 	case "v1":
-		sClient, err := openstack.NewBlockStorageV1(os.provider, gophercloud.EndpointOpts{
-			Region: os.region,
-		})
+		sClient, err := os.NewBlockStorageV1()
 		if err != nil || sClient == nil {
```

Comment:
Generally LGTM. Small nit: maybe err != nil is enough. In NewBlockStorageV1(),if err is returned, sClient is nil.
Two similar cases below